titan/features/haart.py

```diff
@@ -99,18 +101,36 @@ def update_agent(self, model: "model.TITAN"):
                     ]
                     num_haart_agents = self.counts[self.agent.race][self.agent.sex_type]
 
-                    if num_haart_agents < (haart_params.prob * num_dx_agents):
+                    # take value from dictionary for cap
+                    if num_haart_agents < (haart_params.cap * num_dx_agents):
                         self.initiate(model)
                 else:
-                    if model.run_random.random() < (
-                        haart_params.prob * model.calibration.haart.coverage
-                    ):
-                        self.initiate(model)
+                    if self.ever and self.agent.location.params.hiv.reinit_haart:
+                        if model.run_random.random() < haart_params.reinit.prob:
+                            self.initiate(model)
+                    else:
+                        # Find enroll probability based on time since diagnosis
```

Comment:
why is the probability time based, but the cap just flat?  also, what's the difference between a cap and a target (prep)?

Reply:
Those are the same thing just named differently... thoughts on which we should go with?

Probability is agent-specific, so agent x has a y% chance of going on haart at that timestep. Cap/target just says "we want this number of agents on haart. Do we need you to start haart to achieve that?" so it's irrespective of how long the agent has been diagnosed. Does that make sense?